README.md

```diff
@@ -23,5 +23,11 @@ This repo provides a simple tool to generate a JSON file for use with [SlicerMor
    python generate_slicer_json.py <source.csv>
    ```
    This will save `segmentation_category_type.json` in your current working directory.
+
+   To choose a different name for your JSON file, run 
+   ```
+   python generate_slicer_json.py <source.csv> <filename>
+   ```
+   This will save `filename.json` in your current working directory.
```

Comment:
```suggestion
   This will write the JSON output to `filename`, which, if it exists already, will be overwritten. The name may include a path to write it to a place other than the current working directory.
```